# Ball position (x, y)
float64 pos_x
float64 pos_y

# Ball velocity (x, y)
float64 vel_x
float64 vel_y
